symfony-pack/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM quay.io/continuouspipe/symfony-php7.1-nginx:latest
```

Comment:
Can the ":latest" be an ARG to let the branch builds work?

Reply:
Yup, changed 👍